include/dlaf/solver/triangular/mc/triangular_LLN.h

```diff
@@ -65,6 +65,192 @@ void triangular_LLN(blas::Diag diag, T alpha, Matrix<const T, Device::CPU>& mat_
+                                  util::size_t::mul(tile_size.rows(), tile_size.cols()));
+                              Tile<T, Device::CPU> tile(tile_size, std::move(mem_view),
+                                                        tile_size.rows());
+                              comm::sync::broadcast::receive_from(index,
+                                                                  comm_wrapper().rowCommunicator(),
+                                                                  tile);
+                              return std::move(tile);
+                            }),
+                            k_rank_col, mat_a.tileSize(GlobalTileIndex(k, k)), serial_comm());
+        }
+      }
+    }
+
+    for (SizeType j_local = 0; j_local < b_local_cols; ++j_local) {
+      auto j = distr_b.globalTileFromLocalTile<Coord::Col>(j_local);
+
+      if (mat_b.rankIndex().row() == k_rank_row) {
+        auto k_local_row = distr_b.localTileFromGlobalTile<Coord::Row>(k);
+
+        auto kj = LocalTileIndex{k_local_row, j_local};
+
+        // Triangular solve of the k-th row Panel of B
+        hpx::dataflow(executor_hp, hpx::util::unwrapping(tile::trsm<T, Device::CPU>), Left, Lower,
+                      NoTrans, diag, alpha, kk_tile, std::move(mat_b(kj)));
+
+        // Broadcast Bkj column-wise
+        // Avoid useless communication if one-column communicator and if on the last column
+        if (col_comm_size > 1 && k != (mat_b.nrTiles().rows() - 1)) {
+          hpx::dataflow(executor_mpi, hpx::util::unwrapping([](auto&& tile, auto&& comm_wrapper) {
+                          comm::sync::broadcast::send(comm_wrapper().colCommunicator(), tile);
+                        }),
+                        mat_b.read(kj), serial_comm());
+        }
+        panel[j_local] = mat_b.read(kj);
+      }
+      else {
+        if (col_comm_size > 1 && k != (mat_b.nrTiles().rows() - 1)) {
+          panel[j_local] =
+              hpx::dataflow(executor_mpi,
+                            hpx::util::unwrapping([](auto index, auto&& tile_size,
+                                                     auto&& comm_wrapper) -> Tile<const T, Device::CPU> {
+                              memory::MemoryView<T, Device::CPU> mem_view(
+                                  util::size_t::mul(tile_size.rows(), tile_size.cols()));
+                              Tile<T, Device::CPU> tile(tile_size, std::move(mem_view),
+                                                        tile_size.rows());
+                              comm::sync::broadcast::receive_from(index,
+                                                                  comm_wrapper().colCommunicator(),
+                                                                  tile);
+                              return std::move(tile);
+                            }),
+                            k_rank_row, mat_b.tileSize(GlobalTileIndex(k, j)), serial_comm());
+        }
+      }
+
+    }  // j_local loop
```

Comment:
Please avoid empty lines between braces (only exception that an empty line may be useful: to separate functions/classes closing brace from namespace closing braces).
Please also remove closing loop comments, since it is clear from the indentation what a brace closes.